03-Node-npm-modules/app.js

```diff
@@ -0,0 +1,62 @@
+(function e(t,n,r){function s(o,u){if(!n[o]){if(!t[o]){var a=typeof require=="function"&&require;if(!u&&a)return a(o,!0);if(i)return i(o,!0);var f=new Error("Cannot find module '"+o+"'");throw f.code="MODULE_NOT_FOUND",f}var l=n[o]={exports:{}};t[o][0].call(l.exports,function(e){var n=t[o][1][e];return s(n?n:e)},l,l.exports,e,t,n,r)}return n[o].exports}var i=typeof require=="function"&&require;for(var o=0;o<r.length;o++)s(r[o]);return s})({1:[function(require,module,exports){
+var Artist = function(name, awards){
+	this.name = name;
+	this.awards = awards || [];
+};
+
+Artist.prototype.recognice = function(){
+	console.log(this.name + "has the following awards: ");
+	for(award in this.awards){
+		console.log(award);
+	};
+};
+
+Artist.prototype.set = function(attr, values){
```

Comment:
This `set` method does not seems to be OK, try to create the set method as you've already created and add a new one to add all the awards that you expect to be added, this seems to be a mistake due to the awards thing. Right?